upgrade/scripts/sls/csm_can_to_chn/sls_updates.py

```diff
@@ -55,15 +56,30 @@ def sls_and_input_data_checks(
         "Checking input values and SLS data for proper logic.",
         fg="bright_white",
     )
+    can_subnet = networks.get("CAN").ipv4_network()
+    cmn_subnet = networks.get("CMN").ipv4_network()
 
+    if can_subnet.overlaps(chn_subnet[1]):
+        click.secho(
+            f"CAN subnet {can_subnet} overlaps with CHN subnet {chn_subnet[1]}",
+            fg="red",
+        )
+        sys.exit(1)
+
+    if cmn_subnet.overlaps(chn_subnet[1]):
+        click.secho(
+            f"CMN subnet {can_subnet} overlaps with CHN subnet {chn_subnet[1]}",
+            fg="red",
+        )
+        sys.exit(1)
     chn = networks.get("CHN")
     if chn is not None:
         click.secho(
             "    INFO: A CHN network already exists in SLS.",
             fg="white",
         )
     if bican_name == "CHN":
-        if chn_data[1] == ipaddress.IPv4Network("10.104.7.0/24"):
+        if chn_subnet[1] == ipaddress.IPv4Network("10.104.7.0/24"):
```

Comment:
It scares me to have a default subnet here.  If they miss/ignore this warning and it moves on with the default, there will most likely be issues and it might take a while to figure out where the issue is.   It seems like we should instead fail if they don't provide that value on input.